app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -494,8 +503,17 @@ val MIGRATION_12_13 =
         }
     }
 
+val MIGRATION_13_14 =
+    object : Migration(13, 14) {
+        override fun migrate(db: SupportSQLiteDatabase) {
+            db.execSQL(
+                "alter table AppSettings add column key_width INTEGER NOT NULL default 0",
```

Comment:
Based on some of the logic below, 0 is being used as a sub for null. Really this should be just 

`alter table AppSettings add column key_width Integer`

which should make it of type `Int?`

Reply:
Makes complete sense indeed, I'll do that. Sorry I'm new to Kotlin and my work is based on copy-paste of existing code :grimacing: Thanks for your patience :slightly_smiling_face:

Reply:
No probs!